ctail.nim.cfg

```
@@ -1,3 +1,2 @@
 --threads:on
---define:"faststreams_async_backend=std"
-
+--define:"async_backend=std"
```

```suggestion
--define:"asyncBackend=asyncdispatch"
```
https://nim-lang.org/docs/asyncdispatch.html#multiple-async-backend-support